guides/common/modules/proc_destroying-virtual-machines-on-host-delete.adoc

```diff
@@ -0,0 +1,34 @@
+[id="destroying-virtual-machines-on-host-delete"]
+= Destroying virtual machines on host delete
+
+By default, when you delete the provisioned profile host from {Project}, {Project} does not destroy the actual VM on the external compute resource.
+You can configure {Project} to destroy the VM when deleting the host entry on {Project}.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-destroying-virtual-machines-on-host-delete[].
+
+.Prerequisite
+* Your {Project} account has a role that grants the `view_settings` and `edit_settings` permissions.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Administer* > *Settings* > *Provisioning*.
+. Change the value of the *Destroy associated VM on host delete* setting to *Yes*.
+
+[id="cli-destroying-virtual-machines-on-host-delete"]
+.CLI procedure
+* Configure this behavior by using Hammer:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer settings --name destroy_vm_on_host_delete \
+--value true \
+--location "_My_Location_" \
+--organization "_My_Organization_"
+----
+
+[role="_additional-resources"]
+.Next steps
+* You can delete VMs and {Project} destroys them in the compute resource.
+
+[role="_additional-resources"]
+.Additional resources
+* If you do not destroy the associated VM and attempt to create a new VM with the same resource name later, it will fail because that VM name already exists in the external compute resource.
+You can still re-register the existing VM to {Project}.
```

Comment:
I'd say this is more of an admonition (note preferably) than an additional resource.